### PR TITLE
Configmaps params

### PR DESCRIPTION
**Pull request by abhi16394**
# Description
This PR adds missing parameters in configmaps while installing a driver.
This PR was created due to a defect that was raised where dell-csm-operator-controller-manager-* pod from the dell-csm-operator namespace reconciles and overwrites a configmap in the other namespace where I have my csm-isilon-controller- CSI drivers running issue.
In csm-operator,  customers cant add any new parameters into the configmaps. the operator will reconcile and overwrite the configmap everytime the customer edited the configmaps. The parameters that are added were provided in helm and were missing in csm-operator. There were discrepency in parameters provided in configmaps to customers while deploying through helm and csm-operator. 
Hence added all the missing params and verified the installation.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/1193 |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 90% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

image
with resileincy installed which enables podman:
image
with resiliency disabled:
image

### CODE REVIEW

File: tests/e2e/run-e2e-test.sh [hunk 1]
```diff
@@ -39,7 +39,7 @@ cp $CERT_CSI .
 #cp $KARAVICTL /usr/local/bin/
 
 # Copy certificates for Observability tests
-cp ../../samples/observability/selfsigned-cert.yaml ./testfiles/observability-cert.yaml
+#cp ../../samples/observability/selfsigned-cert.yaml ./testfiles/observability-cert.yaml
```
Comment (atye) [inline tests/e2e/run-e2e-test.sh:42]: Should this step be commented?

File: pkg/drivers/commonconfig.go [hunk 2]
```diff
@@ -328,6 +328,8 @@ func GetUpgradeInfo(ctx context.Context, operatorConfig utils.OperatorConfig, dr
 // GetConfigMap get configmap
 func GetConfigMap(ctx context.Context, cr csmv1.ContainerStorageModule, operatorConfig utils.OperatorConfig, driverName csmv1.DriverType) (*corev1.ConfigMap, error) {
 	log := logger.GetLogger(ctx)
+	var podmanLogFormat string
```
Comment (atye) [inline pkg/drivers/commonconfig.go:331]: Are there tests in commonfig_test.go that can be updated or added to reflect these changes?

File: tests/e2e/run-e2e-test.sh (hunk 1 above)
Comment (jooseppi-luna) [reply tests/e2e/run-e2e-test.sh:42]: we should delete lines 41 and 42 imo

**Review by jooseppi-luna — CHANGES_REQUESTED** [review]
can we add the fixed configmap to the n-1, n-2 versions as well? Have you been able to test the install of Unity, PowerFlex, and PowerScale?

File: pkg/drivers/commonconfig.go (hunk 2 above)
Comment (abhi16394) [reply pkg/drivers/commonconfig.go:331]: there are tests for getting the configmaps
nothing to be added more in this section

File: tests/e2e/run-e2e-test.sh (hunk 1 above)
Comment (abhi16394) [reply tests/e2e/run-e2e-test.sh]: deleted the lines

**Comment by abhi16394** [comment]
> can we add the fixed configmap to the n-1, n-2 versions as well? Have you been able to test the install of Unity, PowerFlex, and PowerScale?

I have added the fixed configmap to n-1, n-2 version as well
I was able to test it on powerscale and powerflex

**Review by atye — DISMISSED** [review]

**Review by bjiang27 — APPROVED** [review]

**Review by atye — APPROVED** [review]

**Review by jooseppi-luna — APPROVED** [review]

**Pull request merged**